ocs_ci/ocs/node.py

```diff
@@ -1464,3 +1456,65 @@ def wait_for_new_osd_node(old_osd_node_names, timeout=180):
     except TimeoutExpiredError:
         log.warning(f"New osd node didn't appear after {timeout} seconds")
         return None
+
+
+def add_new_node_and_label_upi_lso(
```

Comment:
the function allows getting multiple nodes. How about reflecting this in its name?